### DISCUSSION TITLE
Docs site currently down?

### DISCUSSION THREAD

**Discussion by aprilmintacpineda** [js-fns/date-fns#3470] [General] [answered]
![image](https://github.com/date-fns/date-fns/assets/21032419/7d5e9e8e-e854-418f-9f56-94a1bae87eba)

I've been trying to access docs site in the past few minutes, it seems to be down at the moment?

**Comment by on3iro** [comment]
jup: https://github.com/date-fns/date-fns/issues/3469

**Comment by Yelsnik** [comment]
I tried it too but it is also down

**Comment by MohdRashid74** [comment]
i tried also many time but still showing "loading problem "please give me any sugestion??

**Comment by joaopedrodcf** [comment, marked as answer]
It's not down anymore I think this discussion can be closed 👍 

https://date-fns.org/docs/Getting-Started

<img width="1512" alt="image" src="https://github.com/date-fns/date-fns/assets/11476152/fa831ab1-6766-47fa-88c4-4caf0ae0b225">
